contracts/cwd-core/src/msg.rs

```diff
@@ -139,78 +144,95 @@ pub enum ExecuteMsg {
-    /// `AdminNominationResponse`.
+    /// Get's the currently nominated admin (if any).
+    #[returns(AdminNominationResponse)]
     AdminNomination {},
-    /// Gets the contract's config. Returns Config.
+    /// Gets the contract's config.
+    #[returns(Config)]
     Config {},
     /// Gets the token balance for each cw20 registered with the
     /// contract.
+    #[returns(Cw20BalanceResponse)]
     Cw20Balances {
         start_after: Option<String>,
         limit: Option<u32>,
     },
     /// Lists the addresses of the cw20 tokens in this contract's
     /// treasury.
+    #[returns(Vec<Addr>)]
     Cw20TokenList {
         start_after: Option<String>,
         limit: Option<u32>,
     },
     /// Lists the addresses of the cw721 tokens in this contract's
     /// treasury.
+    #[returns(Vec<Addr>)]
     Cw721TokenList {
         start_after: Option<String>,
         limit: Option<u32>,
     },
     /// Dumps all of the core contract's state in a single
     /// query. Useful for frontends as performance for queries is more
-    /// limited by network times than compute times. Returns
-    /// `DumpStateResponse`.
+    /// limited by network times than compute times.
+    #[returns(DumpStateResponse)]
     DumpState {},
     /// Gets the address associated with an item key.
+    #[returns(GetItemResponse)]
     GetItem { key: String },
     /// Lists all of the items associted with the contract. For
     /// example, given the items `{ "group": "foo", "subdao": "bar"}`
     /// this query would return `[("group", "foo"), ("subdao",
     /// "bar")]`.
+    #[returns(Vec<String>)]
     ListItems {
         start_after: Option<String>,
         limit: Option<u32>,
     },
     /// Gets all proposal modules associated with the
-    /// contract. Returns Vec<ProposalModule>.
+    /// contract.
+    #[returns(Vec<ProposalModule>)]
     ProposalModules {
         start_after: Option<String>,
         limit: Option<u32>,
     },
     /// Gets the active proposal modules associated with the
-    /// contract. Returns Vec<ProposalModule>.
+    /// contract.
+    #[returns(Vec<ProposalModule>)]
     ActiveProposalModules {
         start_after: Option<String>,
         limit: Option<u32>,
     },
     /// Returns information about if the contract is currently paused.
+    #[returns(PauseInfoResponse)]
     PauseInfo {},
-    /// Gets the contract's voting module. Returns Addr.
+    /// Gets the contract's voting module.
+    #[returns(Addr)]
     VotingModule {},
-    /// Returns all SubDAOs with their charters in a vec
-    /// start_after is bound exclusive and asks for a string address
+    /// Returns all SubDAOs with their charters in a vec.
+    /// start_after is bound exclusive and asks for a string address.
+    #[returns(Vec<SubDao>)]
     ListSubDaos {
         start_after: Option<String>,
         limit: Option<u32>,
     },
     /// Implements the DAO Star standard: https://daostar.one/EIP
+    #[returns(DaoURIResponse)]
```

Comment:
This is the way the new `cosmwasm-schema` works.

Reply:
do you mean why we needed to add `DaoURIResponse`? it's because if we keep it as returning `Option<String`, cosmwasm typescript gen generates a `Nullable_String` type, but then wrongly annotates this as returning `NullableString` for some reason

Reply:
let me know if anyone knows what's up with that

Reply:
@pyramation any thoughts?

Reply:
the latest `cosmwasm/ts-codegen` supports CosmWasm 1.1 — things should work out of the box. 

just make sure to remove the old schemas since it expects a single JSON. 

finally, if there are any issues just let me know we'll take care of them.